java/java.lsp.server/build.xml

```diff
@@ -51,4 +51,65 @@
     <target name="clean" depends="projectized.clean">
         <delete dir="${lsp.build.dir}" />
     </target>
+    <target name="build-vscode-ext" depends="build-lsp-server" description="Build the Visual Studio Code extension.">
+        <exec executable="npm" failonerror="true" dir="vscode">
+            <arg value="install" />
+        </exec>
+
+        <exec executable="npm" failonerror="true" dir="vscode">
+            <arg value="run" />
+            <arg value="compile" />
+        </exec>
+
+        <delete dir="${build.dir}/vscode" />
+        <exec executable="cp" failonerror="true">
+            <arg value="-r" />
+            <arg value="vscode/" />
+            <arg value="${build.dir}/" />
+        </exec>
+
+        <delete dir="${build.dir}/bundles" />
+        <mkdir dir="${build.dir}/bundles/package" />
+        <copy file="vscode/package.json" todir="${build.dir}/bundles/package" />
+
+        <exec executable="mvn" failonerror="true" dir="${nb_all}/nbbuild/misc/prepare-bundles">
+            <arg value="install" />
```

Comment:
Shouldn't package be enough? `exec:java` should use the build package from target, not the one from the local repository.